employees/tests/test_views.py

```diff
@@ -1,9 +1,11 @@
+# pylint: disable=C0302
```

Comment:
That's actually good warning. Too big files are hard to be maintainable. Please create test file with more meaningful name then just `test_views`